kafka/integration_test.go

```diff
@@ -747,6 +551,36 @@ func TestProducerFuncDR(t *testing.T) {
 		})
 }
 
+// test producer with bad messages
+func TestProducerWithBadMessages(t *testing.T) {
+	if !testconfRead() {
```

Comment:
You dont need a broker for these tests, so skip the testconf dependency so it runs on travis.